package.json

```diff
@@ -33,6 +33,7 @@
     "expo-linking": "1.0.1",
     "expo-splash-screen": "0.2.3",
     "expo-web-browser": "8.2.1",
+    "formik": "^2.1.5",
```

Comment:
Make sure to pin your dependencies! Here's a decent article on it

https://docs.renovatebot.com/dependency-pinning/